2차/윤형진_같은-숫자는-싫어.py

```diff
@@ -0,0 +1,20 @@
+def solution(arr):
+    answer = []
+    isBreaked = False
+    comp_idx = 1
+    idx = 0
+    
+    while True : 
+        if idx + 1 >= len(arr):
+            break
+        
+        if arr[idx] == arr[idx + 1] :
+            del arr[idx + 1]
+            
+        else :
+            idx = idx + 1
```

Comment:
continue를 사용해서 구조를 분리하는게 좋은 것 같습니다.

```suggestion
        if arr[idx] == arr[idx + 1] :
            del arr[idx + 1]
            continue
            
        idx = idx + 1
```